src/services/recovery/recovery-state.ts

```diff
@@ -72,21 +87,17 @@ export const getRecoveryState = async ({
   safeAddress: string
   provider: JsonRpcProvider
 }): Promise<RecoveryState[number]> => {
-  const { blockHash } = await _getSafeCreationReceipt(rest)
-
-  const transactionAddedFilter = delayModifier.filters.TransactionAdded()
+  const { blockNumber } = await _getSafeCreationReceipt(rest)
```

Comment:
Let's move this into `queryAddedTransactions` so it doesn't call `_getSafeCreationReceipt` when no queued transactions exist.